cfgov/unprocessed/js/modules/Analytics.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+var Analytics = {
+
+  tagManagerIsLoaded: false,
+
+  EVENT_CATERGORY: 'CFGOV Event',
+
+  /**
+   * Initialize the Analytics module.
+   */
+  init: function() {
+    // detect if Google tag manager is loaded
+    if ( window.hasOwnProperty( 'google_tag_manager' ) ) {
+      Analytics.tagManagerIsLoaded = true;
+    } else {
+      var _tagManager;
+      Object.defineProperty( window, 'google_tag_manager', {
+        enumerable: true,
+        configurable: true,
+        get: function() {
+          return _tagManager;
+        },
+        set: function( value ) {
+          _tagManager = value;
+          Analytics.tagManagerIsLoaded = true;
+        }
+      } );
+    }
+  },
+
+  /**
+   * @name sendEvent
+   * @kind function
+   *
+   * @description
+   * Pushes an event to the GTM dataLayer.
+   *
+   * @param {string} action Name of event.
+   * @param {string} label DOM element label.
+   * @param {Function} callback Function to call on GTM submsission.
+   * @param {number} timeout Callback invocation fallback time.
+  * @returns {Analytics} An instance
+   */
+  sendEvent: function( action, label, callback, timeout ) {
+    var dataLayerOptions = {
+      event:        Analytics.EVENT_CATERGORY,
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2308/commits/fbcb63f708e0ebc537601faeb0357d7e18e23b44